style/javascript.md

```diff
@@ -592,6 +592,91 @@ if (markResolved) {
 
 It's also important to note that Promises do not throw exceptions. If you wish to catch an exception you must explicitly attach a `.catch()` callback to it and listen for the error.
 
+#### Use `fetch()` instead of `$.ajax`/`$.get`/`$.post`/`$.getJSON`
+
+We now provide a polyfill for the [`fetch()` method](https://developer.mozilla.org/en-US/docs/Web/API/GlobalFetch/fetch). We should use this for all Ajax-style requests. Note that in order to use the method you'll need to explictly import it using:
+
+```
+var fetch = require("khan-fetch");
+```
+
+As we wrap the method to ensure that Khan Academy-specific logic is intact (such as handling API Action Results).
+
+##### `$.get()`
+
+Get some JSON data (same use case as `$.getJSON`).
+
+```
+fetch("/some.json")
+    .then((response) => response.json())
+    .then((json) => {
+        // Use the JSON data...
+    })
```

Comment:
In the interests of preserving vertical space, can you make these `then`s and `catch`s all on one line (like you do with the `response => response.json()` line)?